File: morning_sandwich/morning.py
def main():
    case_num = int(input())
    for _ in range(case_num):
        b, c, h = input().split()
        b = int(b)
        c = int(c)
        h = int(h)
        if b <= c + h:
            print(2 * b - 1)
        else:
            print(2 *(c + h) + 1)
    
    
if __name__ == "__main__":
    main()

File: morning_sandwich/morning.c
#include <stdio.h>

int main()
{
    int case_num;
    scanf("%d", &case_num);
    for (int i = 0; i < case_num; i++)
    {
        int b, c, h;
        scanf("%d%d%d", &b, &c, &h);
        if (b <= c + h)
        {
            printf("%d\n", 2 * b - 1);
        }

        else
        {
            printf("%d\n", 2 * (c + h) + 1);
        }
    }
}
